### DISCUSSION TITLE
Clarify standards for vendor inclusion criteria.

### DISCUSSION THREAD

**Discussion by austinlparker** [open-telemetry/opentelemetry.io#4416] [Content Refactoring]
In regards to https://opentelemetry.io/ecosystem/vendors/, there have been some questions arising about what exactly 'Distribution' and 'Native OTLP' mean. 

Upon investigation, vendor support is clarified in the specification @ https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/vendors.md; However, this does not align with the columns on that page. That said, this offers some possible paths:

- Update the vendor page to align with the specification (remove 'distribution' and 'native otlp', change to 'supports opentelemetry' and 'implements opentelemetry')
- Update the specification to add greater specificity in categories
- A secret, third option

**Comment by cartermp** [comment]
I would propose adding the following to "supports opentelemetry"

- By supporting OTLP ingest on their backend directly
- By supporting an OpenTelemetry SDK distribution

And then also updating the table to have this split, if appropriate.

**Comment by cartermp** [comment]
While we're at it, we should also decide if we consider something as "supporting opentelemetry" if they still require the use of a non-otel component in the middle, like an agent that supports OTLP but emits their vendor-specific data format.
